### PR TITLE
day 5 typo check

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_9.txt [hunk 1]
```diff
@@ -407,7 +407,7 @@ I cut the lines of the three corpses.
 [sha:5f83d51db966f1fdb3b10211bffc725374796d14]{
 -- Page 581, Offset 10183.
 -- 胴体をそれぞれ三分割。死体たちはバターを切るように割かれ、死亡する。
-Each of them are split into three pieces. Like a hot knife through butter, the corpses are split open and perish.
+Each of them is split into three pieces. Like a hot knife through butter, the corpses are split open and perish.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Arcueid/Day 5/05_00_ARC05_9.txt:410]: isnt singular supposed to be used for each regardless of the "of phrase" next to it?

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 2]
```diff
@@ -1113,7 +1113,7 @@ This may well be the first time that this kind of thing has ever happened to her
 [sha:ea8805400d9a092fc796bebc4001b1eec27bf4b2]{
 -- Page 128, Offset 8894.
 -- 俺の今の手持ちは……現金で二千円……かぁ……。
-The cash I have on me comes to a grant total of... 2,000 yen...
+The cash I have on me comes to a grand total of... 2,000 yen...
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:1116]: professor shiki operating on grant money to support arcueid

File: script/Arcueid/Day 5/05_00_ARC05_9.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9.txt:410]: you can get away with 'are', but 'is' is more idiomatic

File: script/Arcueid/Day 5/05_00_ARC05_9_3.txt [hunk 3]
```diff
@@ -183,7 +183,8 @@ It's not that Vlov didn't %{i}want%{/i} to approach us.
 [sha:54bfa73523fa85dc6b44ace59002f8df421a35e7]{
 -- Page 813, Offset 10866.
 -- だというのに、俺はヤツに追い風を与えてしまった。
-And in spite of that that, I gave him a tailwind.
+// did that that get past the lint?
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9_3.txt:186]: not sure what you're asking here

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_9.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Arcueid/Day 5/05_00_ARC05_9.txt:410]: which do you prefer then?

File: script/Arcueid/Day 5/05_00_ARC05_9_3.txt (hunk 3 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 5/05_00_ARC05_9_3.txt:186]: it's "how did that that get past the lint". I ruined the joke by forgetting the first word

Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_9_3.txt:186]: second one has a comma on it, so it's a different 'word'

**Review by rschlaikjer — APPROVED** [review]

**Pull request merged**